subsystems/tpc/macros/draw_calo.C

```diff
@@ -0,0 +1,19 @@
+#include <qahtml/OnlProdClient.h>
+#include <qahtml/TPCDraw.h>
+
+R__LOAD_LIBRARY(libonlprodcalo.so)
+
+void draw_calo(const char *rootfile) {
+
+  OnlProdClient *cl = OnlProdClient::instance();
+  /* cl->Verbosity(1); */
+  OnlProdDraw *ex = new TPCDraw();
```

Comment:
Chris made some changes to rename this to `QADraw` - can you do a git pull and retest to make sure everything still works